oliviaruizknott/timer-rails
# Report screen beginnings

A simple report screen!
![screen shot 2018-07-04 at 11 06 44 pm](https://user-images.githubusercontent.com/26205183/42300544-fbd98836-7fde-11e8-9a43-f5b339a17f7b.png)

## To Test
1. Start this meeting: https://salvadortimer.herokuapp.com/meetings/80
2. Advance through the items at your leisure.
3. Note that after clicking "End Meeting", the button text will change to "View Report." Click that.
4. If you went long, you should see some red text and a message about late-ness and lost time. If you ended early, you should see some green text and a message about earliness and saving time.
5. You can click the arrows next to the first number to increase or decrease the attendee count. This will change the collective minute number.
